client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```diff
@@ -19,6 +21,7 @@ import { ApplicationTags } from "../application-tags";
 export interface IApplicationDetailDrawerProps
   extends Pick<IPageDrawerContentProps, "onCloseClick"> {
   application: Application | null;
+  task: Task | undefined | null;
```

Comment:
I wonder if we should keep the `ApplicationDetailDrawer` logic a little more presentational, like maybe have `isTagsTabLoading` and `isReportsTabLoading` props here instead of passing the task all the way in? I'm not 100% sure if that's actually better, so I'm fine either way.